DataFormats/Provenance/interface/ProductHolderIndexHelper.h

```diff
@@ -54,6 +54,7 @@ ProductRegistry is frozen.
 #include "FWCore/Utilities/interface/ProductHolderIndex.h"
 #include "FWCore/Utilities/interface/ProductKindOfType.h"
 #include "FWCore/Utilities/interface/TypeID.h"
+#include "FWCore/Utilities/interface/propagate_const_safe.h"
```

Comment:
I would have prefered to just depend on ```propagate_const.h``` and only include ```propagate_const_safe``` in the cases where the stand alone function is used.